<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>747</width>
    <height>524</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="CalculateButton">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>10</y>
      <width>80</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="AInput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>10</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="CInput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>90</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BInput">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>50</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="Alabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>20</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Blabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>60</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Clabel">
    <property name="geometry">
     <rect>
      <x>240</x>
      <y>100</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>C:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MainLable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>10</y>
      <width>221</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ведите члены квадратичного уравнения:</string>
    </property>
   </widget>
   <widget class="QLabel" name="SituationDisplay">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>50</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Answ1Display">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>80</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="Answ2Display">
    <property name="geometry">
     <rect>
      <x>550</x>
      <y>100</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>20</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="MainLable_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>140</y>
      <width>301</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Введите две стороны треугольника и угол между ними:</string>
    </property>
   </widget>
   <widget class="QLabel" name="ASide">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>180</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>A:</string>
    </property>
   </widget>
   <widget class="QLabel" name="BSide">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>220</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>B:</string>
    </property>
   </widget>
   <widget class="QLabel" name="Andgle">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>260</y>
      <width>21</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>&lt;html&gt;&lt;head/&gt;&lt;body&gt;&lt;p&gt;&lt;span style=&quot; font-family:'arial','sans-serif'; font-size:14px; color:#000000; background-color:#ffffff;&quot;&gt;α&lt;/span&gt;:&lt;/p&gt;&lt;/body&gt;&lt;/html&gt;</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="ASideValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>170</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="BSideValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>210</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="AngleValue">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>250</y>
      <width>113</width>
      <height>31</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="CSideCalculate">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>170</y>
      <width>80</width>
      <height>111</height>
     </rect>
    </property>
    <property name="text">
     <string>Рассчитать</string>
    </property>
   </widget>
   <widget class="QLabel" name="RandomTXT">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>180</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Результат:</string>
    </property>
   </widget>
   <widget class="QLabel" name="CSideValue">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>210</y>
      <width>171</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QRadioButton" name="CheckDegrees">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>240</y>
      <width>69</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Градусы</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="CheckRadians">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>260</y>
      <width>70</width>
      <height>19</height>
     </rect>
    </property>
    <property name="text">
     <string>Радианы</string>
    </property>
   </widget>
   <widget class="QLabel" name="RandomTXT_2">
    <property name="geometry">
     <rect>
      <x>190</x>
      <y>220</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Угол в радианах!</string>
    </property>
   </widget>
   <widget class="QPlainTextEdit" name="InputWindow">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>330</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextEdit" name="OutputWindow">
    <property name="geometry">
     <rect>
      <x>460</x>
      <y>330</y>
      <width>261</width>
      <height>151</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="AddButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>330</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Добавить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="OverrideButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>380</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Заменить</string>
    </property>
   </widget>
   <widget class="QPushButton" name="EditAddButton">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>430</y>
      <width>141</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>&quot;Рандомный&quot; текст</string>
    </property>
   </widget>
   <widget class="QLabel" name="RandomTXT_3">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>310</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ввод</string>
    </property>
   </widget>
   <widget class="QLabel" name="RandomTXT_4">
    <property name="geometry">
     <rect>
      <x>570</x>
      <y>310</y>
      <width>71</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Нечто</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>747</width>
     <height>20</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
